README.md

```diff
@@ -26,6 +26,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
+ * NodeJS 8.0 or higher
  * [Composer](https://getcomposer.org/)
 
 _Composer should be run with --no-dev option unless you are an active LORIS developer._
```

Comment:
Ok to change this to --: ? 
Note: `composer install` should be run with the `--no-dev` option unless you are an active LORIS developer.

Reply:
I think it is ok for nodejs 8.